concepts/05 Widgets/Form/10 Organize Simple Items/10 In Tabs/20 Configure the Tab Panel.md

```diff
@@ -60,11 +60,118 @@ For displaying tabs, the **Form** uses the [TabPanel](/concepts/05%20Widgets/Tab
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxForm :form-data="employee">
+            <DxSimpleItem data-field="firstName" />
+            <DxSimpleItem data-field="lastName" />
+            <DxTabbedItem>
+                <DxTabPanelOptions
+                    :height="250"
+                    :on-title-click="tabPanelTitleClick" />
+                <DxTab title="Info">
+                    <DxSimpleItem data-field="hireDate" />
+                    <DxSimpleItem data-field="city" />
+                </DxTab>
+                <DxTab title="Contacts">
+                    <DxSimpleItem data-field="phone" />
+                    <DxSimpleItem data-field="email" />
+                </DxTab>
+            </DxTabbedItem>
+        </DxForm>
+    </template>
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DxForm, DxSimpleItem, DxTabbedItem, DxTab, DxTabPanelOptions } from 'devextreme-vue/form';
+
+    const employee = {
+        firstName: 'John',
+        lastName: 'Heart',
+        hireDate: new Date(2012, 4, 13),
+        city: 'Los Angeles',
+        phone: '+1(213) 555-9392',
+        email: 'jheart@dx-email.com'
+    };
+
+    export default {
+        components: {
+            DxForm, DxSimpleItem, DxTabbedItem, DxTab, DxTabPanelOptions
+        },
+        data() {
+            return {
+                employee
+            };
+        },
+        methods: {
+            tabPanelTitleClick(e) {
+                // ...
+            }
+        }
+    };
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { Form, SimpleItem, TabbedItem, Tab, TabPanelOptions } from 'devextreme-react/form';
+
+    const employee = {
+        name: 'John Heart',
+        hireDate: new Date(2012, 4, 13),
+        city: 'Los Angeles',
+        phone: '+1(213) 555-9392',
+        email: 'jheart@dx-email.com'
+    };
+
+    class App extends React.Component {
+        constructor() {
+            super();
+            this.tabPanelTitleClick = this.tabPanelTitleClick.bind(this);
+        }
+
+        render() {
+            return (
+                <Form formData={employee}>
+                    <SimpleItem dataField="firstName" />
+                    <SimpleItem dataField="lastName" />
```

Comment:
There are no `firstName` and `lastName` in the data object.

Reply:
Fixed